web/pom.xml

```diff
@@ -76,6 +76,18 @@
               <workingDirectory>${project.build.directory}</workingDirectory>
             </configuration>
           </execution>
+
+          <execution>
```

Comment:
Does this execution happen in `License check / License` step of CI?

Reply:
No.The `License check / License` step will check the license for the `web module`. This is only executed during the `mvn install`.

Reply:
@tiny-dust, could this execute in `License check / License` step? I mean that this execution should be included in `License check / License` step instead of other step.

Reply:
> Does this execution happen in `License check / License` step of CI?

Look this.
![image](https://github.com/apache/incubator-celeborn/assets/49502875/f1651fb1-13ef-45a0-8058-a77417a61f69)

Reply:
> @tiny-dust, could this execute in `License check / License` step? I mean that this execution should be included in `License check / License` step instead of other step.

I understand what you mean, are you suggesting that we add an additional step to execute this command during the License check phase?

Reply:
@tiny-dust, apache-rat-plugin does not check licenses of the frontend files. I suggest the introduced license checker of frontend could be executed in `License check / License` of CI.